gimme

```diff
@@ -153,7 +158,14 @@ _checkout() {
 # _extract "file.tar.gz" "dir"
 _extract() {
 	mkdir -p "${2}"
-	tar -xf "${1}" -C "${2}" --strip-components 1
+
+	if [[ "${1}" == *.tar.gz ]] ; then
+		tar -xf "${1}" -C "${2}" --strip-components 1
+	else
+		unzip -q "${1}" -d "${2}"
+		mv "${2}"/go/* "${2}"
```

Comment:
Yeah :/ we could use `find -maxdepth 1` or something here... The problem stems from `unzip` not having an equivalent `--strip-components` flag.

I googled around and found the below link, which has a similar approach. One of the answers talks about `dotglob`, but I've never used that before and I wasn't sure how much trouble I wanted to get up to... what do you think?

https://superuser.com/questions/518347/equivalent-to-tars-strip-components-1-in-unzip